org.claws_mail.Claws-Mail.json

```diff
@@ -256,26 +257,48 @@
       ]
     },
     {
-        "name": "pinentry",
-        "config-opts": [
-            "--enable-pinentry-gtk2",
-            "--disable-ncurses",
-            "--disable-fallback-curses",
-            "--disable-pinentry-curses",
-            "--disable-pinentry-emacs",
-            "--disable-pinentry-gnome3",
-            "--disable-pinentry-qt5",
-            "--disable-pinentry-tty",
-            "--disable-pinentry-tqt",
-            "--disable-pinentry-fltk"
-        ],
-        "sources": [
-            {
-                "type": "archive",
-                "url": "https://www.gnupg.org/ftp/gcrypt/pinentry/pinentry-1.1.0.tar.bz2",
-                "sha256": "68076686fa724a290ea49cdf0d1c0c1500907d1b759a3bcbfbec0293e8f56570"
-            }
-        ]
+      "name": "pcsc-lite",
```

Comment:
This should be on host iirc

Reply:
That was my first thought too. I tried for weeks to get that to work :-P. However, we need a stripped down version in the container if we want to work with the opened 'pcsc' socket.

I took the example from https://github.com/flathub/org.mozilla.Thunderbird/blob/master/org.mozilla.Thunderbird.json#L96